app/graphql/base_service.rb

```diff
@@ -1,10 +1,16 @@
 class BaseService
 
-  attr_accessor :params, :object, :user
+  attr_accessor :params, :model_id, :user
 
-  def initialize(params: {}, object: nil, object_id: nil, user: nil)
+  # def initialize(params: {}, object: nil, object_id: nil, user: nil)
+  #   @params = params.to_h.symbolize_keys
+  #   @object = object || model.find_by(id: object_id)
+  #   @user = user
+  # end
+
+  def initialize(params: {}, model_id: nil, user: nil)
     @params = params.to_h.symbolize_keys
-    @object = object || model.find_by(id: object_id)
+    @model_id = model_id
```

Comment:
Just curious could you please explain why do you change `object` to `model_id` and call `model.find(model_id)` on each services?

Reply:
when we use the object with `find_by(object_id)`, we can't check the exception for the object(record).
If you see my latest commit, you can check exceptions on services.